package.json

```diff
@@ -95,8 +95,10 @@
   },
   "dependencies": {
     "@resin.io/valid-email": "^0.1.0",
+    "@types/bluebird": "3.5.25",
```

Comment:
How about bumping the devDependency instead of adding this one and having both?

Reply:
Oh, I had not noticed that `@types/bluebird` was already listed in the devDependencies section. `@types/bluebird` is relevant for typescript code, and we use typescript in production not just in development (e.g. not just for running tests), so I assume that we want to have it in dependencies section. We should not duplicate it in both sections, of course, so my action was to move it devDependencies to dependencies. Is there a reason not to do this?

Reply:
I can answer my own question... `@types/bluebird` is not needed for a "production" installation to be installed and work correctly, and the fewer required packages in production, the better. So I have moved it back to the devDependencies section. :-)